src/main/kotlin/io/emeraldpay/dshackle/rpc/NativeCall.kt

```diff
@@ -481,17 +482,24 @@ open class NativeCall(
             ReaderData(ctx.upstream, ctx.upstreamFilter, ctx.callQuorum, signer, tracer),
         )
         val counter = reader.attempts()
+        val isRipple = ctx.upstream.getChain() in listOf(Chain.RIPPLE__MAINNET, Chain.RIPPLE__TESTNET)
```

Comment:
Can we add comment here about the reasons of stream disabling ? For next generations :)